pacman-nodejs-app/tools/migrate/awzkmt.sh

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env bash
```

Comment:
Did you make any changes to the `kmt.sh` tool to create this new script? We want to extend the existing `kmt.sh` script instead of creating a new one.